ajax_with_jquery_exercise/script.js

```diff
@@ -0,0 +1,94 @@
+$(function () {
+    var $ol = $("ol");
+
+    // login aside
+    $(".login").on("click", function (e) {
+        e.preventDefault();
+        $("aside").toggle();
+    });
+
+    // login functionality
+    $(".login-submit").on("click", function (e) {
+        e.preventDefault();
```

Comment:
could you abstract this logic into its own function so you could reduce the duplication below?